lib/iris/analysis/__init__.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2016, Met Office
+# (C) British Crown Copyright 2010 - 2017, Met Office
```

Comment:
I don't understand this stuff either.

We have recently seen merge do weird things.
Very much as you said "that file hasn't been touched..." see [this example](#2283).
In that case, the merged result seemed to contain the travis.yml (with its associated iris-test-data SHA) from the (older) PR branch instead of the (newer) upstream target branch, ***despite the fact that the PR does not touch that file.***
@marqh and I gave up trying to understand why + how, and he just rebased and made a new branch.